flink-formats/flink-orc-nohive/pom.xml

```diff
@@ -63,7 +63,7 @@ under the License.
 			<version>${orc.version}</version>
 			<classifier>nohive</classifier>
 			<exclusions>
-				<!-- Exclude ORC's Hadoop dependency and pull in Flink's shaded Hadoop. -->
+				<!-- Exclude ORC's Hadoop dependency and pull in Flink's Hadoop. -->
```

Comment:
let's remove the exclusions just like in flink-orc, and then the explicit hadoop dependencies